content/en/docs/client-options.md

```diff
@@ -30,6 +30,10 @@ the Python [acme](https://github.com/certbot/certbot/tree/master/acme) module is
 
 {{< /clients >}}
 
+# ACMEv1 End of Life
+
+We encourage subscribers to select a client that implements ACMEv2 support. Per our API announcement on the [community forum](https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1), we will eventually stop allowing new domains to validate via ACMEv1 followed eventually by completely disabling the ACMEv1 protocol.
```

Comment:
Can you attach the link to the whole sentence [Per our announcement on the community forum] instead? Makes it more visible, and makes it clearer that it's not just a generic link.